README.md

```diff
@@ -1,4 +1,4 @@
-### :warning: Guard is [looking for new maintainers](https://groups.google.com/forum/#!topic/guard-dev/2Td0QTvTIsE). Please [contact me](mailto:thibaud@thibaud.gg) if you're interested.
+### :exclamation::exclamation: Guard is [currently accepting more maintainers/contributors](https://groups.google.com/forum/#!topic/guard-dev/2Td0QTvTIsE). Please [contact me](mailto:thibaud@thibaud.gg) if you're interested in joining the team. :exclamation::exclamation:
```

Comment:
@e2 What about adding your email for the contact me link, I guess it would be more useful no?